### DISCUSSION TITLE
Can't redirect output to log file

### DISCUSSION THREAD

**Discussion by qpirsel** [Chia-Network/chia-blockchain#5182] [Q&A]
Hi,

when I try to redirect the output of e.g. chia plots check with  `>>` in CMD or  `|tee` in Powershell I don't get any output file. As a cross-check other commands like `dir` work as intended.

For testing purposes I redirected stderr and the I got an output file but with strange additional sequences in the file.

![Snipaste_2021-05-16_22-23-56](https://user-images.githubusercontent.com/17856684/118411554-6f61e100-b695-11eb-8a44-5d9f77c8a086.png)

Any hints to solve this/my problem?

**Comment by lvcivs** [comment]
Yeah, it seems it's printing to `stderr` for some reason. 

On my Linux CLI the word INFO is shown in green, I think the strange character sequence marks the start/end of the color highlighting.

**Reply by qpirsel** [reply]
That sounds sensible. But I really wonder why this isn't mentioned in the offical FAQ. There are "tips" how to generate logs under Windows CLI but has anyone ever tested it or was it only an assumption that this "should" work.

Anyway, if nobody comes with a solution within the next time I'm going to open a bug report.

**Reply by loppefaaret** [reply]
*>> will direct both err and out
